frontend/bank-transfer-ui/src/app/login/login.component.html

```diff
@@ -1,11 +1,15 @@
-<div class="card bg-custom">
-  <div class="card-body">
-    <h1 class="logo">Bank<span class="brand-color">UI</span></h1>
-    <form action="/main">
-      <div class="p-2"><input class="form-control form-custom form-control-lg" name="username" type="text" placeholder="username" required></div>
-      <div class="p-2"><input class="form-control form-custom form-control-lg" name="password" type="password" placeholder="password" required></div>
-      <div class="p-2"><button type="submit" class="btn btn-custom btn-lg"><span class="text-uppercase">Login</span></button></div>
-    </form>
+<div class="login-custom-bg">
+  <div class="container">
+    <div class="card bg-custom">
+      <div class="card-body">
+        <h1 class="logo">Bank<span class="brand-color">UI</span></h1>
+        <form action="/main">
+          <div class="p-2"><input class="form-control form-custom form-control-lg" name="username" type="text" placeholder="username" required></div>
+          <div class="p-2"><input class="form-control form-custom form-control-lg" name="password" type="password" placeholder="password" required></div>
+          <div class="p-2"><button type="submit" class="btn btn-custom btn-lg"><span class="text-uppercase">Login</span></button></div>
+        </form>
+      </div>
+    </div>
   </div>
 </div>
 <app-footer></app-footer>
```

Comment:
Jeśli app-footer jest na każdej stronie (tj. login i main), to wrzuć go do index.html

Reply:
Nie ładuje footera, gdy dodaję go do index.html

Reply:
Jeśli angular Ci go nie ładuje, to zakładam, że footer jest zagnieżdżony w module main, a powinien być globalnie w app

Reply:
Też próbowałem go tam wyrzucić, musiałbyś podejść.